### PR TITLE
Windows: Change dll search order

### PR DESCRIPTION
**Pull request by kbengs**

### CODE REVIEW

**Comment by kbengs** [comment]
libffi is moved because it is loaded before SetDefaultDllDirectories (caused by ctypes import)
If it is not in the lib folder Windows will search Windows system folders and then all locations in PATH 
and finally find it in pdfarranger.exe folder. (this is when using openwith) (if mitmproxy is installed it will find libffi there)

With SetDefaultDllDirectories(0x1000) C:\Windows\System and C:\Windows will not be searched (as it does before this change). I don't know if that can be a problem. On my machine it is not.

Any idea what to do about the except other than add a comment?

File: pdfarranger/pdfarranger.py [hunk 1]
```diff
@@ -15,6 +15,14 @@
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
 import os
+import ctypes
+
+if os.name == 'nt':
+    try:
+        ctypes.windll.kernel32.SetDefaultDllDirectories(0x1000)
+    except (ImportError, AttributeError):
```
[the remark this answers is not in the document]
Comment (dreua) [reply pdfarranger/pdfarranger.py:23]: Is this expected to throw exceptions regularly? Otherwise I'd say its best to print the error.

**Comment by jeromerobert** [comment]
As far as I understand the only case when an exception could be triggered in this try/except is for an unpatched Windows 7 and older. As we only distribute the 64bit version of PDF Arranger, it is very unlikely that this will happen. So I think a comment is enough.

Appart from that I think the PR is ready for merge.

**Comment by kbengs** [comment]
Printing to terminal will not Work on Windows so I just added a comment

**Comment by dreua** [comment]
> Printing to terminal will not Work on Windows so I just added a comment

Of course 🤦‍♂️ 
I'm just too deep into thinking Linux...

Thank you for the great work, looking forward to seeing less random Windows issues here!

**Pull request merged**